docs/cloud.md

```diff
@@ -0,0 +1,59 @@
+# Deploying Antrea on a cloud provider
+
+Antrea may run in networkPolicyOnly mode in some cloud managed clusters. This document describes
+ steps to create EKS using terraform.
+
+## Common Pre-requisites
+1. To run EKS cluster, install and configure aws cli, see
```

Comment:
```suggestion
1. To run EKS cluster, install and configure aws cli (either version 1 or 2), see
```
since you tested with version 1 and I tested with version 2

Reply:
done.